server.js

```diff
@@ -72,14 +76,39 @@ function initialiseProxy (app) {
   proxy.use()
 }
 
-function initialiseAppVariables (app) {
-  app.set('view engine', 'html')
-  app.set('vendorViews', path.join(__dirname, '/govuk_modules/govuk_template/views/layouts'))
-  app.set('views', path.join(__dirname, '/app/views'))
-}
-
 function initialiseTemplateEngine (app) {
-  app.engine('html', require(path.join(__dirname, '/lib/template-engine.js')).__express)
+  // Define app views
+  const appViews = [
+    path.join(__dirname, '/govuk_modules/govuk_template/views/layouts'),
+    path.join(__dirname, '/app/views-nunjucks')
+  ]
+
+  // Configure nunjucks
+  // see https://mozilla.github.io/nunjucks/api.html#configure
+  const nunjucksConfiguration = {
+    express: app, // the express app that nunjucks should install to
+    autoescape: true, // controls if output with dangerous characters are escaped automatically
+    throwOnUndefined: false, // throw errors when outputting a null/undefined value
+    trimBlocks: true, // automatically remove trailing newlines from a block/tag
+    lstripBlocks: true, // automatically remove leading whitespace from a block/tag
+    watch: false, // reload templates when they are changed (server-side). To use watch, make sure optional dependency chokidar is installed
+    noCache: false // never use a cache and recompile templates each time (server-side)
+  }
+  if ((!nodeEnv) || (nodeEnv !== 'production')) {
```

Comment:
should perhaps just be in the nunjuncs configuration:
```
nocache: NODE_ENV !== 'production',
watch: NODE_ENV !== 'production
```

Reply:
Also if you inline these you could inline the whole config as you wouldn't need to declare `const nunjucksConfiguration`. You could probable inline the appViews then too.